Improve docs for get_file_list (#666)
astronomer/astro-sdk: docs/astro/files/operators/get_file_list.rst

When to use the ``get_file_list`` operator
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can use ``get_file_list`` retrieve a list of available files based on a storage path and an Airflow connection. This can be useful for dynamically generating tasks based on which files are available on your storage system.

Reviewer: ```suggestion
You can use ``get_file_list``to retrieve a list of available files based on a storage path and the Airflow connection. Based on the files available on your system storage, this can generate tasks dynamically.
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> We can use ``get_file_list`` when you want a list file available in storage based on storage path pattern and Airflow connection. This might useful when you want to create dynamic task on the response of this operator.

Reviewer 2: ```suggestion
You can use ``get_file_list`` retrieve a list of available files based on a storage path and an Airflow connection. This can be useful for dynamically generating tasks based on which files are available on your storage system.
```

> The supported filesystem are :ref:`file_location`

Reviewer 2: ```suggestion
The supported filesystems are :ref:`file_location`
```

> The supported filesystem are ``Local, S3, GCS, and HTTP``.

Reviewer 3: Don't we have a rst file containing all the supported filesystems? like we have for supported databases

Author: Not sure. I can see we have a supported technologies section in README https://github.com/astronomer/astro-sdk#supported-technologies. Referring to this here would misguide.  Any thought on how we can improve README and refer here?

Reviewer 4: @pankajastro you can use - https://github.com/astronomer/astro-sdk/blob/main/docs/supported_file.rst

Author: fixed 9b436a9190564e4067c3f643859d5834c83223fb

> The below example, get a file list from the GCS bucket and upload them to the Bigquery table by creating a dynamic number of parallel tasks using the dynamic task map ``expand`` method.

Reviewer 5: ```suggestion
In the below example, get a file list from the GCS bucket and upload them to the Bigquery table by creating a dynamic number of parallel tasks using the dynamic task map ``expand`` method.
```

> The below example, get a file list from the GCS bucket and upload them to the Bigquery table by creating a dynamic number of parallel tasks using the dynamic task map ``expand`` method.

Reviewer 2: ```suggestion
The following example retrieves a file list from the GCS bucket and dynamically generates tasks using ``expand`` to upload each listed file to a Bigquery table.
```
